src/shared/util/field_default_constants.h

```diff
@@ -32,18 +32,37 @@ class FieldCircularArc;
 namespace FieldConstantsRoboCup2018A {
     const double kFieldLength = 12000.0;
     const double kFieldWidth = 9000.0;
-    const double kGoalWidth = 1200.0;
+    const double kGoalWidth = 1800.0;
     const double kGoalDepth = 180.0;
     const double kGoalHeight = 155.0;
-    const double kBoundaryWidth = 250.0;
+    const double kBoundaryWidth = 300.0;
     const double kLineThickness = 10.0;
-    const double kPenaltyAreaDepth = 1200.0;
-    const double kPenaltyAreaWidth = 2400.0;
+    const double kPenaltyAreaDepth = 1800.0;
+    const double kPenaltyAreaWidth = 3600.0;
     const double kCenterCircleRadius = 500.0;
     const double kGoalLineToPenaltyMark = 8000.0;
     const double kBallRadius = 21.5;
     const double kMaxRobotRadius = 90.0;
     const int kNumCamerasTotal = 2;
-    const int kNumCamerasLocal = 2;
+    const int kNumCamerasLocal = 1;
```

Comment:
Just to make sure, this means the number of camera per ssl-vision, right? So, we use two computers for vision at Div A field, one for each camera? (I just remember the Div B setup from Sydney was 1 computer and 1 camera)

Reply:
correct. Local in this case means, on the current computer/ssl-vision instance. For division A, we use two NUC computers, each having one camera. The parameters influence the generation of the camera calibration control points.